doc/extending/creating_an_op.rst

```diff
@@ -1,40 +1,18 @@
 
 .. _creating_an_op:
 
-Creating a new :class:`Op`: Python implementation
+Creating a new :ref:`op`: Python implementation
```

Comment:
Where do you want the `` :ref:`op` `` to point to? To https://pytensor--1086.org.readthedocs.build/en/1086/extending/op.html or to https://pytensor--1086.org.readthedocs.build/en/1086/extending/graphstructures.html#op?

I think many of the broken cross-references come from this page: https://pytensor--1086.org.readthedocs.build/en/1086/extending/op.html, which is wildly inconsistent with itself. If the cross-reference would ideally point there we should fix this page first.

Reply:
@OriolAbril extending op seems much more useful than the graphstructures#op, so I guess there.